frontend/packages/enduser-frontend/src/views/applications/daycare-application/daycare-application.vue

```diff
@@ -1162,7 +1162,10 @@ SPDX-License-Identifier: LGPL-2.1-or-later
           : ''
       },
       minimumStartdate(): null | string {
-        return this.isPreschool
+        // If application is already sent, the preferred start date can only be moved forward
+        return (this.model.status.value !== 'CREATED' && this.model.preferredStartDate) ?
+          this.model.preferredStartDate
```

Comment:
This makes it impossible for the applicant to even select an earlier date than what they have picked. So, when editing, if they select 1.3.2021 at first, they can't change that to 28.2.2021 anymore.

Reply:
Yes they can - they are in CREATED stated until they send the application, thus we use the latter (old check)

Reply:
Got it - once the application is sent, editing the date is only possible forward even before save. Fixed by storing the original preferred start date and using it instead